defaults/main.yml

```diff
@@ -0,0 +1,73 @@
+---
+
+# Main directory for certificates and lego
+lets_encrypt_directory_path: "/etc/ssl"
+
+lets_encrypt_lego_archive_http_path: "https://github.com/go-acme/lego/releases/download/v3.7.0/lego_v3.7.0_linux_amd64.tar.gz"
+
+lets_encrypt_resource_name: "lego_resource_key"
+
+lets_encrypt_account_email: "linux_monitor@sitewards.com"
+
+# number of days before expiry the renewal will be performed
+lets_encrypt_renew_limit: 10
```

Comment:
let's please put 30 days by default, otherwise we are supposed to get alerts all the time.

our monitoring will alert if certs are expiring 30 days ahead

Reply:
done